var/spack/repos/builtin/packages/rust/package.py

```diff
@@ -76,24 +76,22 @@ class Rust(Package):
     # Compiling Rust requires a previous version of Rust.
     # The easiest way to bootstrap a Rust environment is to
     # download the binary distribution of the compiler and build with that.
-    depends_on("rust-bootstrap", type="build")
 
     # Pre-release version dependencies
-    depends_on("rust-bootstrap@beta", type="build", when="@beta")
     depends_on("rust-bootstrap@nightly", type="build", when="@master")
     depends_on("rust-bootstrap@nightly", type="build", when="@nightly")
 
     # Stable version dependencies
-    depends_on("rust-bootstrap", type="build")
-    depends_on("rust-bootstrap@1.59:1.60", type="build", when="@1.60")
-    depends_on("rust-bootstrap@1.64:1.65", type="build", when="@1.65")
-    depends_on("rust-bootstrap@1.69:1.70", type="build", when="@1.70")
-    depends_on("rust-bootstrap@1.72:1.73", type="build", when="@1.73")
-    depends_on("rust-bootstrap@1.73:1.74", type="build", when="@1.74")
-    depends_on("rust-bootstrap@1.74:1.75", type="build", when="@1.75")
-    depends_on("rust-bootstrap@1.77:1.78", type="build", when="@1.78")
-    depends_on("rust-bootstrap@1.80:1.81", type="build", when="@1.81")
+    depends_on("rust-bootstrap@1.84:1.85", type="build", when="@1.85")
     depends_on("rust-bootstrap@1.82:1.83", type="build", when="@1.83")
+    depends_on("rust-bootstrap@1.80:1.81", type="build", when="@1.81")
+    depends_on("rust-bootstrap@1.77:1.78", type="build", when="@1.78")
+    depends_on("rust-bootstrap@1.74:1.75", type="build", when="@1.75")
```

Comment:
1.76 missing.

https://gitlab.spack.io/spack/spack/-/jobs/15264507

Reply:
Should be fixed now.